Public/Sdk/SelfHost/BuildXL/Testing/QTest/qtestFramework.dsc

```diff
@@ -137,6 +137,10 @@ function runTest(args : TestRunArguments) : File[] {
         qTestLogs: logDir,
         tags: args.tags,
         weight: args.weight,
+        qCodeCoverageEnumType: Environment.hasVariable("[Sdk.BuildXL]qCodeCoverageEnumType")

```

Comment:
I would still prefer to add this code directly to Tool.QTestRunner.dsc. So later on if we want to enable this feature for Office, we don't need to come back and change the code but only set the environment variables in their build pipelines.

Reply:
Done.